webpack.config.js

```diff
@@ -22,6 +22,10 @@ module.exports = {
   resolve: {
     extensions: ['.ts', '.js'],
   },
+  node: {
+    // @payid-org/utils has a dependency on fs
+    fs: 'empty',
```

Comment:
let me know if you guys think this is bogus; this is used specifically in https://github.com/payid-org/utils/blob/master/src/verifiable/keys.ts#L17. the functions that operate on files can never work in the browser, so this feels alright to me, although perhaps i should update the documentation...

Reply:
@hbergren is working on removing that dependency because it's only needed by the PayID cli. This looks like an acceptable work around until then.